Directory.Build.targets

```diff
@@ -36,6 +36,16 @@
     <KnownFrameworkReference Remove="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
+  <!-- Global Analyzer Config -->
+  <ItemGroup>
+    <!-- Always include Common.globalconfig -->
+    <EditorConfigFiles Include="$(RepositoryEngineeringDir)config\globalconfigs\Common.globalconfig" />
```

Comment:
I had just kind of assumed based on "is_global" that you could only have one top-level `.editorconfig`. I assume contended properties are broken on a "last imported" basis? Didn't find any documentation on that with a cursory search.

Reply:
> I assume contended properties are broken on a "last imported" basis?

Contended properties cause a build warning (CI error) and are ignored from all sources. If a rule configuration is present in .globalconfig, you know it's only in that one file.